config.json

```diff
@@ -1289,6 +1289,15 @@
         "searching"
       ]
     },
+    {
+      "slug": "rest-api",
+      "uuid": "4c89b6a8-0e89-46c9-5a7e-31bfe6a57007",
+      "core": false,
+      "unlocked_by": null,
+      "difficulty": 0,
```

Comment:
What do you consider the difficulty of this exercise to be? To me, the difficulty is mainly due to keeping an administration of users and dealing with JSON. My gut feeling would put it at difficulty 5 or 6, due to the relative large amount of code one has to write (compared to other exercises).

Reply:
Not exactly sure of the difficulty compared to different exercise. When I look at Markdown exercise, I would probably make Rest Api difficulty a little bit harder than markdown, so a 6 seems good.